build/Linux+BSD/mscore.1.in

```diff
@@ -339,6 +339,8 @@ internal file sanity check log (JSON)
 MPEG Layer III (lossy compressed audio)
 .It Li mpos
 measure positions (XML)
+.It Li mposx
+measure positions in pixel unit (XML)
```

Comment:
Maybe it would make sense to document that those pixels are at 360(?) DPI, to make it possible for user to convert them to inches or any other units? Without knowing this `mposx`/`sposx` formats don't seem to be much different from the current `mpos`/`spos`.

Reply:
Hi @dmitrio95 ! That's indeed the default value configured in musescore but it should be PPI then, as DPI don't relate to pixel distance (unless used as if PPI, but that's conceptually wrong). 
You can read more there: https://musescore.org/en/node/311914 (my bad, posted initially the wrong link in the PR comment ^^)

Reply:
> I would prefer to have "absolute"/"pure" coordinates on one side, and derive those factors from the actual image file I'm working with if needed.

But why are 360 PPI coordinates better than 100 or any other PPI? Internal PPI (or DPI, the terms [seem to be reversed](https://github.com/musescore/MuseScore/blob/5e88871beb594d90f8abfffa3d30a4151af166d1/libmscore/mscore.h#L86-L88) in internal MuseScore terminology) is not necessarily related to PPI of the exported images as resolution of exported images is actually configurable by user (see settings in Edit→Preferences→Export dialog). Therefore, if I am correct, if these files are intended to be useful in general case there should be some way to convert those coordinates to something else, otherwise the new format doesn't seem to make the situation any better. Or am I missing something here?

Reply:
Normally, changing dpi shouldn't change any resolution of an image, but indeed changing ppi will change the resolution. When I change DPI in musescore, the resolution indeed change, but shouldn't, that's really misleading for comprehension to use one concept as the other (even tho, frequent), maybe 'another issue to solve' ? I've mentioned that parameter on the forum, but thought it was "badly used" only for the coordinates export and didn't see the obvious that it relates to png resolution (so ppi) and not the png dpi (which I thought, and wouldn't makes sense neither in fact as "dpi"), thanks for the questioning, you're on point! As-is, indeed, it's working only with "default values", thanks!
So now, from what I understand, the formula:
`ndpi = (((qreal) preferences.getDouble(PREF_EXPORT_PNG_RESOLUTION) / DPI) * 12.0)`
Could be seen as: 
`ndpi = (pixel_scaling_factor_regarding_output_res * 12) `
All in all, if aware of how it works internally for DPI/PPI, the remaining question I have is the factor 12 applied, as actual mpos/spos coordinates can relate to a png export distance if divided by 12. So perhap's writting documentation about this could helps, along with understanding that 12 factor ? "fixing" the actual mpos/spos as is.
Also, maybe a better implementation for mposx/sposx would be a fractional value ? (so was my proposal in the previous comment, I phrased it badly indeed, my bad). That would remove most of those factors that would make spos/mpos dependent of specific export settings? 
I had a feedback from the user that used this functionality and that's how they handle the positions on their side (fractionnal [0,1]), that seems like a good flexible solution for me, what is your opinion about that ?

Thanks for you inputs ! waiting for your opinions :)

(and for dpi/ppi, if needed, but not sure tho, just by precaution, and maybe others reading this, it's way better explained than anything I would write and shortly here: https://www.sony.com/electronics/support/articles/00027623 )

Reply:
Hi @dmitrio95, don't want to seem insinstent or impatient, but really interesting in your point point of view :) If you think I should discuss this on the forum, let me know ^^

Reply:
The problem with these DPI is that they are a *user preference* (and therefore not stable)…

Reply:
@dmitrio95  Actually, for the new format (mposx/mpx), they are in whatever DPI was used to create the PNG, as they map 1:1 to pixels in the PNG exported during the same run (with `-j`) or a run with the same configuration.

Reply:
@mirabilos yep, but imho, that's an issue as they are dependent on the knowledge of the user settings. For instance, if I have a png from a user with specific settings and *pos files from another, those will not match. I'm really getting on the side of a fractionnal value as those are totally independant of most settings, meaning way more flexible. Waiting for the @dmitrio95 input on that, but imho, will implement fractionnal values, that seems way more legit, flexible, elegant and future proof. (For instance, using a today *pos file with wathever you export in the future, delegating to the consumer the match between wathever he choose as image export and the position files)

Reply:
> Actually, for the new format (mposx/mpx), they are in whatever DPI was used to create the PNG

As far as I understand, DPI settings for PNG export are not taken into account in `mposx` in the current version of this pull request. Having those values directly correspond to PNG pixels would indeed be reasonable. But SVG or PDF export seem to use different DPI so this solution won't be applicable to those types of export with arbitrary DPI setting for PNG export (current `.mpos` also isn't).

The solution which seems most logical to me is the following:
1) Use pixels corresponding to PNG export DPI, like @mirabilos suggests;
2) Add page size information to the same file so users who need to scale these values to something else will also have the necessary information.

This way would require a bit more changes to the format (to add a new XML tag with page dimensions) but would probably make the new format more or less useful for a general use case.

Reply:
@dmitrio95 Thanks for your feedback ! Agree on that regarding the old format, will do like that then. Will update the existing format to give the page dimension, totally fine personally with that. What about the 12 factor applied? My first guess would be to just apply it too to the page dimensions, as my goal is also to provide the less breaking changes needing code updates for consumers.
Also that would require scaling for user who don't work on the same dimension, or at least redivide by the legacy factor 12 in place, but all in all, the fractionnal possible one too, might also be less code and functionnal scope to maintain over time to not include a new one if the old is fixed, even tho, producing fractional value is quite straight forward. But yep, will "rollback" on those changes, at the end, my goal is just to be able too to consume those files (which is solved for me since few days), but if I can make like easier for others as well ^^

I'm also trying to solve https://musescore.org/en/node/312022 related to the same feature.
If you have any pointer on how to get the bbox of a Chord element (currently (0,0,0,0)), started few minutes ago to test  min/maxing positions of child elements but that seems like a lot of hastle for something that should mostly be working by default. (cfr issue for more info) If you have any experience with that, it would be more than welcomed and highly appreciated :)